src/containerapp/azext_containerapp/tests/latest/test_containerappjob_with_yaml.py

```diff
@@ -18,6 +19,7 @@
 
 
 class ContainerAppJobsExecutionsTest(ScenarioTest):
+    @serial_test() #  The region where the resources are located is not the default TEST_LOCATION. Parallel running will affect each other, so serial running is required.
```

Comment:
why not in default region will influence tests

Reply:
When the tests running, they configure and use the location from `az configure`.
If the value is not the default TEST_LOCATION, they will influence tests which running in TEST_LOCATION when running at the same time.